spec/classes/nginx_spec.rb

```diff
@@ -36,26 +38,29 @@
         it { is_expected.to contain_nginx__resource__streamhost('streamhost1').with_proxy('streamproxy') }
       end
 
-      context 'nginx::package' do
+      context 'nginx::package' do # rubocop:disable RSpec/EmptyExampleGroup
```

Comment:
Once https://github.com/rubocop/rubocop-rspec/pull/1249 is released, this should no longer be needed